cmd/scollector/collectors/ifstat.go

```diff
@@ -0,0 +1,13 @@
+package collectors
+
+import "regexp"
+
+var ifstatRE = regexp.MustCompile(`\s+(eth\d+|em\d+_\d+/\d+|em\d+_\d+|em\d+|` +
```

Comment:
If you do the registerInit thing, this file can be moved back into ifstat_linux.go